### DISCUSSION TITLE
就是我用学校的移动校园网都可以翻墙，但是回到老家用电信宽带就不能上网，哟反复试过。我觉得是电信网络的问题，但是又不知道怎么解决，我用手机的移动热点给电脑供网可以翻墙。但是换电信的wifi就不能翻墙。这是怎么回事。

### DISCUSSION THREAD

**Discussion by shilun12** [2dust/v2rayNG#3333] [General]
对的

**Comment by shilun12** [comment]
有没有大佬帮我出出点子

**Comment by gydchenxiao** [comment]
不会用的Cloudflare中转的节点吧，现在电信对cf官方ip一点都不友好，运营商里就属电信对cf官方ip延迟最高。

**Comment by yux666** [comment]
节点测速都可以 
就是启动服务失败 
该怎么自行解决 ![Screenshot_20241030_180202.jpg](https://github.com/user-attachments/assets/b547e05f-4301-4e4e-a3f3-f0f231ed0427)

![Screenshot_20241030_180143.jpg](https://github.com/user-attachments/assets/1f86c37f-b092-49c9-b685-c7fb3687a77c)
